packages/node/test/e2e/withdrawals.feature.ts

```diff
@@ -10,7 +10,7 @@ it('processes withdrawals only once', async () => {
 
   const provider = e2e.buildProvider();
 
-  const requests = [fixtures.operation.buildWithdrawal({ requesterId: 'alice', destination: 'alice' })];
+  const requests = [fixtures.operation.buildWithdrawal({ requesterId: 'alice' })];
```

Comment:
This is different to `fixtures.requests.buildWithdrawal()`. Here it builds a request in the format intended for the `operation` package here:

https://github.com/api3dao/airnode/blob/78ee8da7339fc0f4b111ce72230f523a90053abf/packages/operation/src/config/evm-dev-config.json#L75-L80

The withdrawal isn't a valid request object in the `node` package. But I think it would actually be better to expose these fixture functions (currently in `node/test/fixtures/operation/`) from the `operation` package, rather than the `node` package.

Reply:
doh! you are right. The expected objects have different attributes.

I'll create a separate issue for moving everything under `node/test/fixtures/operation/ `to the `operation` package.